extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryKit.java

```diff
@@ -54,10 +57,12 @@ public class WindowOperatorQueryKit implements QueryKit<WindowOperatorQuery>
 {
   private static final Logger log = new Logger(WindowOperatorQueryKit.class);
   private final ObjectMapper jsonMapper;
+  private final QueryContext queryContext;
```

Comment:
I would just store the flag is WindowedOperationTranformEnabled no ?
There is no need to store the entire context as a class level object. I would go one level further and just pass the flag from outside. Ie 
`new WindowOperatorQueryKit(context.jsonMapper(),isGlueingTranformEnabled()))` so that queryKits never see the "high level context" and all logic is contained in the controller impl.

Reply:
Makes sense, have made the change.